lib/caesar.rb

```diff
@@ -1,6 +1,107 @@
+require 'pry'
 class Caesar
-  attr_reader :key
+  attr_reader :cipher
 
   def initialize
+    @cipher = []
   end
-end
\ No newline at end of file
+
+     plain = "the quick brown fox jumps over the lazy dog"
+     shift = 3
+
+
+    def eng_to_cipher(string, shift)
+      alphabet = Array("a".."z")
+      encryption_key = Hash[alphabet.zip(alphabet.rotate(shift))]
+      cipher = string.chars.map do |key|
+        encryption_key.fetch(key," ")
+      end.join
+    end
+
+    def eng_to_cipher_case_sensitive(string, shift)
```

Comment:
No need for a case sensitive and case insensitive. Just make one that is case sensitive.